updatecli/updatecli.d/jdk21.yaml

```
@@ -32,24 +32,18 @@ sources:
           to: _
 
 conditions:
-  checkTemurinAlpineDockerImage:
-    kind: dockerimage
-    name: Check if the container image "eclipse-temurin:<lastVersion>-jdk-alpine" is available
-    disablesourceinput: true
+  checkTemurinAlpineRelease:
+    name: Check if the "<lastVersion>" is available for alpine
+    kind: shell
     spec:
-      image: eclipse-temurin
-      tag: '{{source "lastVersion" }}-jdk-alpine'
-  checkTemurinDebianDockerImages:
-    kind: dockerimage
-    name: Check if the container image "eclipse-temurin:<lastVersion>-jdk-focal" is available
+      command: sh ./jdk-download-url.sh '{{source "lastVersion" }}' alpine "x64"
     disablesourceinput: true
+  checkTemurinLinuxRelease:
+    name: Check if the "<lastVersion>" is available for Linux
+    kind: shell
     spec:
-      architectures:
-        - amd64
-        - arm64
-        - s390x
-      image: eclipse-temurin
-      tag: '{{source "lastVersion" }}-jdk-focal'
+      command: sh ./jdk-download-url.sh '{{source "lastVersion" }}' standard "x64,aarch64,s390x"
```

Same result here as mentioned earlier.  It worked better for me in my interactive testing when I used a space separator between the ARCHS values rather than using a "," separator.
```suggestion
      command: sh ./jdk-download-url.sh '{{source "lastVersion" }}' standard "x64 aarch64 s390x"
```